docs/react-testing-library/api.md

```diff
@@ -252,34 +246,6 @@ expect(firstRender).toMatchDiffSnapshot(asFragment())
 
 ---
 
-## `cleanup`
```

Comment:
I removed this full section as it didn't seem pertinent anymore.I think there may be an argument for keeping it here though, just to show how to use `afterEach(cleanup)`